### ISSUE TITLE
Builds failing with 'unsupported custom section: '.debug_info''

### ISSUE THREAD

**Issue by dhardy** [koute/cargo-web#116]
Recent Rand builds seem to be failing with this error:
```
$ cargo --list | egrep "^\s*web$" -q || cargo install cargo-web
$ cargo web test --target wasm32-unknown-unknown --nodejs --features=stdweb
warning: debug builds on the wasm32-unknown-unknown are currently totally broken
         forcing a release build
   Compiling <...>
    Finished release [optimized] target(s) in 2m 39s
    Garbage collecting "rand-dfadd5273a3b9286.wasm"...
    Processing "rand-dfadd5273a3b9286.wasm"...
thread 'main' panicked at 'unsupported custom section: '.debug_info'', src/wasm_context.rs:646:25
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```
([Example](https://travis-ci.org/rust-lang-nursery/rand/jobs/403390519))

Any idea what's up? (It could be due to outdated versions of `cargo-web` installed in our CI cache due to avoiding the update, I suppose.)

**Comment by kirillt** [comment]
I have the same on clean Ubuntu 17.10 VM.
* cargo-web 0.6.13
* stdweb 0.4.7
* rust nightly

**Comment by koute** [comment]
Thanks for the report! Should be fixed in `0.6.14`.
